composer/trainer/trainer.py

```diff
@@ -608,39 +606,10 @@ class Trainer:
                 For fine-grained control on checkpoint saving (e.g. to save different types of checkpoints
                 at different intervals), leave this parameter as ``None``, and instead pass
                 instance(s) of :class:`~.CheckpointSaver` directly as ``callbacks``.
-        save_filename (str, optional): A format string describing how to name checkpoints.
```

Comment:
Can the `checkpoint_save_path` docstrings mention something about additional options using the `CheckpointSaver` callback? Otherwise, users may think we have a pretty limited interface.

Reply:
they do right?

Reply:
this isn't a subtraction. It's added back in below.